src/main/resources/assets/gtnhlib/models/blocks/model_test_tint_mul.json

```
@@ -0,0 +1,56 @@
+{
+	"format_version": "1.21.6",
+	"credit": "Made with Blockbench",
+	"parent": "block/cube",
+	"textures": {
+		"particle": "planks_oak",
+		"down": "planks_oak"
+	},
+	"elements": [
+		{
+			"from": [0, 0, 0],
+			"to": [8, 16, 16],
+			"rotation": {"angle": 0, "axis": "y", "origin": [8, 0, 0]},
+			"faces": {
+				"north": {"uv": [8, 0, 16, 16], "texture": "#down", "tintindex": 3},
+				"east": {"uv": [0, 0, 16, 16], "texture": "#down", "tintindex": 0},
+				"south": {"uv": [0, 0, 8, 16], "texture": "#down", "tintindex": 2},
+				"west": {"uv": [0, 0, 16, 16], "texture": "#down", "tintindex": 1},
+				"up": {"uv": [0, 0, 8, 16], "texture": "#down", "tintindex": 4},
+				"down": {"uv": [8, 0, 16, 16], "texture": "#down", "tintindex": 5}
```

Changed the uv to match vanilla & don't stretch for 8px sections
```suggestion
				"down": {"uv": [0, 0, 8, 16], "texture": "#down", "tintindex": 5}
```